switchbot/adv_parsers/blind_tilt.py

```diff
@@ -0,0 +1,25 @@
+"""Library to handle connection with Switchbot."""
+from __future__ import annotations
+
+
+def process_woblindtilt(
+    data: bytes | None, mfr_data: bytes | None, reverse: bool = True
+) -> dict[str, bool | int]:
+    """Process woBlindTilt services data."""
+
+    if mfr_data is None:
+        return {}
+
+    device_data = mfr_data[6:]
+
+    _tilt = max(min(device_data[2] & 0b01111111, 100), 0)
+    _in_motion = bool(device_data[2] & 0b10000000)
+    _light_level = (device_data[1] >> 4) & 0b00001111
+
+    return {
+        "calibration": bool(data[1] & 0b01000000) if data else None,
```

Comment:
I don't know that this is the calibration flag. This byte is always 0 on my devices. I'm going to investigation further to see if I can figure out if there is another bit being used.

Reply:
Figured it out. It's the first bit of `device_data[1]`

```suggestion
        "calibration": bool(device_data[1] & 0b00000001),
```